packages/siop-oid4vp/lib/types/SIOP.types.ts

```diff
@@ -509,6 +516,12 @@ export interface VerifiedIDToken {
   verifyOpts: VerifyAuthorizationResponseOpts
 }
 
+export interface VerifiedOpenID4VPSubmissionDcql {
+  dcqlQuery: DcqlQuery
+  presentations: WrappedVerifiablePresentation[]
```

Comment:
I feel this is missing the mapping between query and submitted presentations.
Basically what the PEX submission provides. 

So e.g. presentation at index 0 (with credential 1), satisfies the request from dcql query with index 1.

I don't think there's currently a mapping between the dcqlQuery and the presentations right?

Reply:
maybe this should just contain the vp_token structure (so the record) but with wrapped vps?